frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -2529,9 +2529,6 @@ def tlu_cell_input_value(i):
                 resulting_type.bit_width,
             )
             if delta_precision:
-                bit_value = self.reinterpret(
```

Comment:
what happened here? an issue you found when doing this PR, I imagine

Reply:
Yeap, some tests were failing because of this.